tests/kernel_flexible_centerer.py

```diff
@@ -54,16 +56,18 @@ def test_fit_reference(self):
         directly from the equation.
         """
         K = np.random.uniform(0, 100, size=(3, 3))
-        ref_cmean = np.random.uniform(0, 100, size=(3))
-        ref_mean = np.random.uniform(0, 100, size=(1))[0]
+
```

Comment:
In general, this test is written in order to check whether we are working correctly with the reference parameters or not. Now it just duplicates `test_fit_transform`, why is it needed? And yes, now we do not check the work on reference parameters, which is generally important.

Reply:
Good point -- I'll revert the meaning of this test.

Reply:
Ok, great!